packages/universal-crossplane/overlay/questions.yaml

```diff
@@ -0,0 +1,184 @@
+  required: false
+  type: enum
+  default: "edit"
+  options:
+  - "edit"
+  - "view"
+  description: "Permission for Upbound Cloud on the cluster"
+  group: "Upbound Cloud"
+# Basic Crossplane configuration
+- variable: replicas
+  label: replicas
+  description: "The number of replicas to run for the Crossplane pods"
+  type: int
+  default: 1
+  required: true
+  group: "Crossplane"
+# Advanced Crossplane configuration
+- variable: advancedCrossplaneConfiguration
+  description: "Show advanced configuration for Crossplane"
+  label: Show advanced configuration
+  type: boolean
+  default: false
+  show_subquestion_if: true
+  group: "Crossplane"
+  subquestions:
+  - variable: leaderElection
+    label: leaderElection
+    description: "Enable leader election for Crossplane Managers pod"
+    type: boolean
+    default: true
+    required: false
+    group: "Crossplane"
+  - variable: deploymentStrategy
+    label: deploymentStrategy
+    description: "The deployment strategy for the Crossplane and RBAC Manager (if enabled) pods"
+    type: enum
+    default: "RollingUpdate"
+    options:
+    - "RollingUpdate"
+    - "Recreate"
+    required: true
+    group: "Crossplane"
+  - variable: priorityClassName
+    label: priorityClassName
+    description: "Priority class name for Crossplane and RBAC Manager (if enabled) pods"
+    type: string
+    required: false
+    group: "Crossplane"
+  - variable: metrics.enabled
+    label: metrics.enabled
+    description: "Expose Crossplane and RBAC Manager metrics endpoint"
+    type: boolean
+    required: false
+    group: "Crossplane"
+# Basic Crossplane RBAC Manager configuration
+- variable: rbacManager.deploy
+  label: rbacManager.deploy
+  description: "Deploy RBAC Manager"
+  type: boolean
+  default: true
+  required: true
+  group: "Crossplane RBAC Manager"
+- variable: rbacManager.replicas
+  label: rbacManager.replicas
+  description: "The number of replicas to run for the RBAC Manager pods"
+  type: int
+  default: 1
+  required: true
+  group: "Crossplane RBAC Manager"
+# Advanced Crossplane RBAC Manager configuration
+- variable: advancedRBACManagerConfiguration
+  description: "Show advanced configuration for Crossplane RBAC Manager"
+  label: Show advanced configuration
+  type: boolean
+  default: false
+  show_subquestion_if: true
+  group: "Crossplane RBAC Manager"
+  subquestions:
+  - variable: rbacManager.leaderElection
+    label: rbacManager.leaderElection
+    description: "Enable leader election for RBAC Managers pod"
+    type: boolean
+    default: true
+    group: "Crossplane RBAC Manager"
+  - variable: rbacManager.managementPolicy
+    label: rbacManager.managementPolicy
+    description: "The extent to which the RBAC manager will manage permissions. All indicates to manage all Crossplane controller and user roles. Basic indicates to only manage Crossplane controller roles and the crossplane-admin, crossplane-edit, and crossplane-view user roles."
```

Comment:
Suggestion: `RBAC manager permissions. 'All' enables management for every Crossplane controller and user role. 'Basic' enables management just for Crossplane controller roles and the crossplane-admin, crossplane-edit, and crossplane-view user roles.`